lib/Behavior/Path/ResetFilingPathBehavior.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\DocumentManager\Behavior\Path;
+
+use Sulu\Component\DocumentManager\Behavior\Mapping\ParentBehavior;
+
+/**
+ * Automatically positions the document at a configured location as a child of a node
+ * named after the documents alias.
+ *
+ * For example, if you specify the base path to be "/cmf/example" and the document has
+ * the alias "foobar" then the parent will be set to "/cmf/example/foobar".
+ *
+ * If the parent document does not exist, it will be created.
+ */
+interface ResetFilingPathBehavior extends ParentBehavior
```

Comment:
I would say the interfaces `StructureTypeFilingSubscriber` and `AliasFilingSubscriber` should extend this interface, so that a document has not to implement this interface explicitly. Otherwise it would be possible to get the wrong behavior with the chaining of the path you had before.